egs2/aishell/asr1/conf/decode_asr_streaming2.yaml

```diff
@@ -0,0 +1,7 @@
+beam_size: 20
```

Comment:
what is the difference of streaming.yaml and streaming2.yaml?
If either of them is unnecessary, please remove it.

Reply:
the streaming2 is for the previous streaming method (not real streaming). If there is no need to keep it, I can delete it.